### PR TITLE
Update SLES install for tomcat

### PR DESCRIPTION
**Pull request by Mrod1598**
Updated sles install for tomcat to fix an issue where the metric `tomcat.sessions` wasn't being generated due to no default page existing.

### CODE REVIEW

**Comment by Mrod1598** [comment]
Kokoro has already passed 
image

File: integration_test/third_party_apps_data/applications/tomcat/sles/install [hunk 1]
```diff
@@ -10,6 +10,41 @@ Environment="JAVA_OPTS=-Djava.awt.headless=true"
...
+<p><b>tomcat9-docs</b>: This package installs a web application that allows to browse the Tomcat 9 documentation locally. Once installed, you can access it by clicking <a href="docs/">here</a>.</p>
+
+<p><b>tomcat9-examples</b>: This package installs a web application that allows to access the Tomcat 9 Servlet and JSP examples. Once installed, you can access it by clicking <a href="examples/">here</a>.</p>
+
+<p><b>tomcat9-admin</b>: This package installs two web applications that can help managing this Tomcat instance. Once installed, you can access the <a href="manager/html">manager webapp</a> and the <a href="host-manager/html">host-manager webapp</a>.</p>
+
+<p>NOTE: For security reasons, using the manager webapp is restricted to users with role "manager-gui". The host-manager webapp is restricted to users with role "admin-gui". Users are defined in <code>/etc/tomcat9/tomcat-users.xml</code>.</p>
+
+</body>
+</html>
```
[the remark this answers is not in the document]
Comment (Mrod1598) [reply integration_test/third_party_apps_data/applications/tomcat/sles/install:44]: That does seem to work. Updated.

**Comment by jefferbrecht** [comment]
LGTM modulo tests 👍

**Review by jefferbrecht — APPROVED** [review]

**Review by jefferbrecht — APPROVED** [review]

**Comment by Mrod1598** [comment]
> Sorry for the delay in merging, Kokoro was stuck for a bit yesterday. I can merge once the branch is updated.
> 
> I can't remember if folks from ObserveIQ tried this before and the option didn't appear for them, but it might be worth another try: you can [enable maintainers](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/working-with-forks/allowing-changes-to-a-pull-request-branch-created-from-a-fork#enabling-repository-maintainer-permissions-on-existing-pull-requests) to make changes to your pull requests which would let us update the branch before merging.

yeah don't think we can do it for the whole org fork, seems to only be individual forks that that is enabled for.

**Pull request merged**